app/src/main/java/fr/gaulupeau/apps/Poche/network/ImageCacheUtils.java

```diff
@@ -16,6 +16,7 @@
 
 import fr.gaulupeau.apps.InThePoche.BuildConfig;
 import fr.gaulupeau.apps.Poche.App;
+import fr.gaulupeau.apps.Poche.data.Settings;
```

Comment:
this import is not needed, isn't it?

Reply:
I'm not a Java dev specifically so I'm no expert, but App.getSettings() returns a Settings object, and my change includes 2 calls to that; the file previously didn't use any Settings functionality, so I added the import for it.  If that's not the right behavior let me know.

Reply:
Sorry, was confused; thought the file referenced here was StorageHelper.java, not ImageCacheUtils.java.  I didn't make any other changes to this file so I suspect this import is not needed (it does use a Settings object but not in a way that requires the import).  Will remove that change.